src/experiments/atari/Dopamine_DQN_Atari.jl

```diff
@@ -23,7 +23,7 @@ function RLCore.Experiment(
     N_ACTIONS = length(action_space(env))
     init = glorot_uniform(rng)
 
-    create_model() =
```

Comment:
This experiment is kind of special. It is to reproduce the similar result from Dopamine. So I prefer to not change the network structure here.